ops/services/app_service/metabase-readonly/_var.tf

```diff
@@ -0,0 +1,66 @@
+variable "name" {
```

Comment:
See comment below for `ops/services/app_service/metabase-readonly/main.tf` - I think we can get rid of this file entirely.

Reply:
Removed this (and the other files in this folder)